nuxeo-runtime/nuxeo-runtime-stream/src/main/resources/OSGI-INF/avro-service.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="org.nuxeo.runtime.avro" version="1.0">
+
+  <documentation>
+    The component allows to register and use avro schemas.
+  </documentation>
+
+  <implementation class="org.nuxeo.runtime.avro.AvroComponent" />
+
+  <service>
+    <provide interface="org.nuxeo.runtime.avro.AvroSchemaStoreService" />
+    <provide interface="org.apache.avro.message.SchemaStore" />
```

Comment:
It's uncommon (maybe unheard of) in Nuxeo services to expose directly third-party interfaces. We'll have to check if it's not a problem in the future.

Reply:
It's working well and respect the inner design perfectly, I thought it was a good way for any consumer to have the actual interface it needs.